[debug] [2018-10-03T03:17:55.221Z] ----------------------------------------------------------------------
[debug] [2018-10-03T03:17:55.227Z] Command:       /usr/local/bin/node /usr/local/bin/firebase serve
[debug] [2018-10-03T03:17:55.228Z] CLI Version:   4.2.1
[debug] [2018-10-03T03:17:55.228Z] Platform:      linux
[debug] [2018-10-03T03:17:55.228Z] Node Version:  v10.8.0
[debug] [2018-10-03T03:17:57.323Z] Time:          Tue Oct 02 2018 23:17:55 GMT-0400 (GMT-04:00)
[debug] [2018-10-03T03:17:57.323Z] ----------------------------------------------------------------------
[debug] 
[debug] [2018-10-03T03:17:57.349Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform"]
[debug] [2018-10-03T03:17:57.351Z] > authorizing via signed-in user
[debug] [2018-10-03T03:17:57.351Z] [iam] checking project bookme-78399 for permissions ["firebase.projects.get"]
[debug] [2018-10-03T03:17:57.356Z] >>> HTTP REQUEST POST https://cloudresourcemanager.googleapis.com/v1/projects/bookme-78399:testIamPermissions  

[debug] [2018-10-03T03:17:58.634Z] <<< HTTP RESPONSE 200
[debug] [2018-10-03T03:17:58.642Z] >>> HTTP REQUEST GET https://cloudresourcemanager.googleapis.com/v1/projects/bookme-78399  
 
[debug] [2018-10-03T03:17:59.733Z] <<< HTTP RESPONSE 200
[info] 
[info] === Serving from '/mnt/c/Users/Yasnaret/Desktop/Laboratoria/bookMe'...
[info] 
[debug] [2018-10-03T03:18:02.418Z] Error: EACCES: permission denied, mkdir '/usr/local/lib/node_modules/firebase-tools/node_modules/@google-cloud/functions-emulator/logs'
    at Object.mkdirSync (fs.js:729:3)
    at Object.assertLogsPath (/usr/local/lib/node_modules/firebase-tools/node_modules/@google-cloud/functions-emulator/src/emulator/logs.js:65:10)
    at new Controller (/usr/local/lib/node_modules/firebase-tools/node_modules/@google-cloud/functions-emulator/src/cli/controller.js:77:32)
    at FunctionsEmulator.start (/usr/local/lib/node_modules/firebase-tools/lib/functionsEmulator.js:113:21)
    at Object.start (/usr/local/lib/node_modules/firebase-tools/lib/serve/functions.js:9:29)
    at /usr/local/lib/node_modules/firebase-tools/lib/serve/index.js:17:21
    at arrayMap (/usr/local/lib/node_modules/firebase-tools/node_modules/lodash/lodash.js:639:23)
    at Function.map (/usr/local/lib/node_modules/firebase-tools/node_modules/lodash/lodash.js:9556:14)
    at _serve (/usr/local/lib/node_modules/firebase-tools/lib/serve/index.js:15:7)
    at Command._action (/usr/local/lib/node_modules/firebase-tools/commands/serve.js:47:12)
[error] 
[error] Error: An unexpected error has occurred.
